### PR TITLE
doc: update changelog for v1.10.0

### PR DESCRIPTION
**Pull request by QuentinBrosse**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
 ## 1.10.0 (Unreleased)
+
+FEATURES
+
+* provider/scaleway: update terraform SDK to 0.12.0 [GH-127]
```
Comment (jerome-quere) [inline CHANGELOG.md]: why `[GH-127]` and  not `#127`?

Comment (QuentinBrosse) [reply CHANGELOG.md]: I follow the [best practices format](https://www.terraform.io/docs/extend/best-practices/versioning.html#entry-format)

**Comment by DenBeke** [comment]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+IMPROVEMENTS
+
+* provider/scaleway: add .vscode to .gitignore [GH-128]
+
+BUG FIXES:
+
+* provider/scaleway: build command in the README [GH-134]
+* datasource/image: TestAccScalewayDataSourceImage_Filtered [GH-129]
```
Comment (nicolai86) [inline CHANGELOG.md]: nit: we usually only include user facing bug fixes here;

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,19 @@
 ## 1.10.0 (Unreleased)
+
+FEATURES
+
+* provider/scaleway: update terraform SDK to 0.12.0 [GH-127]
+
+IMPROVEMENTS
+
+* provider/scaleway: add .vscode to .gitignore [GH-128]
```
Comment (nicolai86) [inline CHANGELOG.md]: nit: we usually only include customer facing improvements.

Comment (QuentinBrosse) [reply CHANGELOG.md]: Thanks for the feedback, I will fix it

**Pull request merged**